lang/src/org/partiql/lang/ast/ast.kt

```diff
@@ -477,7 +477,7 @@ data class CreateTable(
  * @param keys The expressions that extract the keys from the target to be applied to the index.
  */
 data class CreateIndex(
-    val tableName: String,
+    val tableId: VariableReference,
```

Comment:
Please follow the advice in the comment I've identified below instead of using `VariableReference` here.

The new `Identifier` class should *not* inherit from `ExprNode`, but it should implement `HasMetas`.